app/src/main/java/com/it3048/mobiledeviceproject/dto/Calendar.kt

```diff
@@ -0,0 +1,13 @@
+package com.it3048.mobiledeviceproject.dto
+
+import com.google.gson.annotations.SerializedName
+
+class Calendar {
```

Comment:
1) Is this used anywhere?  It appears to be a new class without a reference.
2) Careful when naming a class "Calendar".  There is already an interface with that same name in the API.